CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - [#5939](https://github.com/influxdata/influxdb/issues/5939): Support viewing and killing running queries.
 - [#6073](https://github.com/influxdata/influxdb/pulls/6073): Iterator stats
 - [#6079](https://github.com/influxdata/influxdb/issues/6079): Limit the maximum number of concurrent queries.
+- [#6075](https://github.com/influxdata/influxdb/issues/6075): Limit the maximum running time of a query.
```

Comment:
I'm a bit confused with how we do this and we should probably standardize it. I've been using the issue that the PR fixes while some include the PR. If there is no issue for the PR, then I include the PR link.